x-pack/plugin/security/src/test/java/org/elasticsearch/integration/MultipleIndicesPermissionsTests.java

```diff
@@ -127,6 +141,64 @@ public void testSingleRole() throws Exception {
         assertHitCount(searchResponse, 1);
     }
 
+    public void testMonitorRestrictedWildcards() throws Exception {
+
+        assertSecurityIndexActive();
```

Comment:
unfortunately this does not do what you want it to do. This will pass if the index does not exist.

Reply:
Good catch!